dm/pkg/conn/utils_test.go

```diff
@@ -59,3 +60,67 @@ func TestGetBinlogDB(t *testing.T) {
 	require.Equal(t, binlogIgnoreDB, "ignore_db")
 	require.Nil(t, mock.ExpectationsWereMet())
 }
+
+func TestGetMasterStatus(t *testing.T) {
+	ctx := context.Background()
+	tctx := tcontext.NewContext(ctx, log.L())
+
+	db, mock, err := sqlmock.New()
+	require.NoError(t, err)
+	baseDB := NewBaseDB(db)
+
+	cases := []struct {
+		rows           *sqlmock.Rows
+		binlogName     string
+		pos            uint32
+		binlogDoDB     string
+		binlogIgnoreDB string
+		gtidStr        string
+		err            error
+		flavor         string
+	}{
+		// For MySQL
+		{
+			sqlmock.NewRows([]string{"File", "Position", "Binlog_Do_DB", "Binlog_Ignore_DB", "Executed_Gtid_Set"}).
+				AddRow("ON.000001", 4822, "", "", "85ab69d1-b21f-11e6-9c5e-64006a8978d2:1-46"),
+			"ON.000001",
+			4822,
+			"",
+			"",
+			"85ab69d1-b21f-11e6-9c5e-64006a8978d2:1-46",
+			nil,
+			gmysql.MySQLFlavor,
+		},
+		// For MariaDB
+		{
+			sqlmock.NewRows([]string{"File", "Position", "Binlog_Do_DB", "Binlog_Ignore_DB"}).
+				AddRow("mariadb-bin.000016", 475, "", ""),
+			"mariadb-bin.000016",
+			475,
+			"",
+			"",
+			"0-1-2",
```

Comment:
Nit: It won't appear in mariaDB
```suggestion
			"",
```

Reply:
I conformed to this example

https://github.com/pingcap/tiflow/blob/c982fca9d18e295c20b826e4f81490007ea96e80/dm/pkg/conn/utils.go#L133-L146

Here its `gtidStr` is `"0-1-2"` in MariaDB.
I have changed my tests according to your suggestion, do I need to modify this comment as well?

Reply:
Sorry, I make a mistake. Current test is correct. Can you revert https://github.com/pingcap/tiflow/pull/7372/commits/1c100ab8fe57c490c246ecf8ff5552a0ccf03b3f? After that we can merge this PR

Reply:
Modified